docs/source/examples/substituting_nonnumeric_values.py

```diff
@@ -0,0 +1,20 @@
+# adapted from t_sub.py / t_NomialSubs
+from gpkit import Variable
+from gpkit.small_scripts import mag
+
+x = Variable("x", "m")
+xvk = x.varkeys.values()[0]
```

Comment:
Line 6 throws:

```
Exception has occurred: TypeError
'dict_values' object does not support indexing
  File "/home/matthew/Desktop/the_gpkit/docs/source/examples/substituting_nonnumeric_values.py", line 6, in <module>
    xvk = x.varkeys.values()[0]
```
which may be because of a Python 2 artifact, where the original function returned a list, and now in Python3 It returns a `view`. One solution is to wrap `list(x.varkets.values())` see: [Link](https://stackoverflow.com/questions/17431638/get-typeerror-dict-values-object-does-not-support-indexing-when-using-python/17431716) . Looking at my debugger it looks like `x.varkeys.values()` returns `None` - is that the expected behavior?

Line 16 than throws:

```
Exception has occurred: TypeError
string indices must be integers
  File "/home/matthew/Desktop/the_gpkit/docs/source/examples/substituting_nonnumeric_values.py", line 16, in <module>
    assert abs(expected - mag(x.sub(x_, y_).c)) < 1e-6
```
Since, `x` is `gpkit.Variable(x [m])`, I think `x.sub(x_, y_)` is invalid because `x` contains a single variable. If I run the code without `y_` it runs fine, until I pass in a string `"x"` into `x.sub(x_)` which throws: 

`x.sub('x') ValueError: could not convert string to float: 'x'`

Needless to say. A lot of things are going on here. I think I may need your insight.